### DISCUSSION TITLE
Force disable ValidatingInterceptor in MnD-JDBC

### DISCUSSION THREAD

**Discussion by garkin** [micronaut-projects/micronaut-data#1032] [Q&A] [answered]
Half of methods of `CrudRepository` and `AsyncCrudRepository` for unknown reason goes through `ValidatingInterceptor`.
It's pretty complex and in my case very useless. I'm using Kotlin and don't care about nullability checks, nor do i have any validating annotations on my pojo-jdbc-entity. Nor even once i've seen any exceptions from it, compiler already does a perfect job. It looks like a cpu and gc waste.

Are there any way to disable it?

`micronaut.validator.enabled: false` doesn't help, it breaks user input validation in other app parts.

**Comment by graemerocher** [comment]
remove `micronaut-validation` from your classpath

**Reply by garkin** [reply]
It's the same as `micronaut.validator.enabled: false`.

**Comment by dstepanov** [comment, marked as answer]
That's because repository methods have `@Valid` annotation. You can create a similar interface without them.
